controllers/index.js

```diff
@@ -1,152 +1,118 @@
 'use strict';
+
+const _ = require('lodash');
 const moment = require('moment');
+
+
+const statements = [
+    'It\'s not too late to start!',
+    'Off to a great start, keep going!',
+    'Half way there, keep it up!',
+    'So close!',
+    'Way to go!',
+    'Now you\'re just showing off!'
+];
+
 /**
  * GET /
  */
 exports.index = (req, res) => {
-    function findPrs(username) {
-        const _ = req.app.get('_');
-        const q = req.app.get('q');
-        const github = req.app.get('github');
-
-        const deferred = q.defer();
-
-        const options = {
-            q: `-label:invalid+created:2017-09-30T00:00:00-12:00..2017-10-31T23:59:59-12:00+type:pr+is:public+author:${username}`
-        };
+    const github = req.app.get('github');
+    const username = req.query.username;
 
-        q.all([
-            github.search.issues(options),
-            github.users.getForUser({ username })
-        ]).then(gitData => {
-            const foundPrs = gitData[0];
-            const user = gitData[1];
-            const prs = [];
-
-            console.log('API calls remaining: ' + user.meta['x-ratelimit-remaining']);
-
-            _.each(foundPrs.data.items, event => {
-                const repo = event.pull_request.html_url.substring(0, event.pull_request.html_url.search('/pull/'));
-
-                const hacktoberFestLabels = _.filter(event.labels, label => label.name.toLowerCase() === 'hacktoberfest');
-
-                const returnedEvent = {
-                    has_hacktoberfest_label: hacktoberFestLabels.length > 0,
-                    number: event.number,
-                    open: event.state === 'open',
-                    repo_name: repo.replace('https://github.com/', ''),
-                    title: event.title,
-                    url: event.html_url,
-                    created_at: moment(event.created_at).format('MMMM Do YYYY'),
-                    user: {
-                        login: event.user.login,
-                        url: event.user.html_url
-                    }
-                };
+    if (!username) {
+        if (req.xhr) {
+            return res.render('partials/error', { layout: false });
+        }
 
-                prs.push(returnedEvent);
-            });
+        return res.render('index');
+    }
 
-            const requests = [];
+    Promise.all([
+        findPrs(github, username),
+        github.users.getForUser({username})
+            .then(logCallsRemaining)
+    ])
+        .then(([prs, user]) => { 
```

Comment:
Possibly due to the version of node I'm on (I want to target latest LTS version) but this reports the following error for me: 

```
/Users/ian/Code/hacktoberfest-checker/controllers/index.js:36
        .then(([prs, user]) => { 
               ^

SyntaxError: Unexpected token [
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:404:25)
    at Object.Module._extensions..js (module.js:432:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at Object.<anonymous> (/Users/ian/Code/hacktoberfest-checker/index.js:16:25)
    at Module._compile (module.js:425:26)
    at Object.Module._extensions..js (module.js:432:10)
error Command failed with exit code 1.
```

Reply:
What version of node do you use?

Reply:
Sorry I just realised I had dipped into version 5 to debug something, works fine on 6+ 👍